services/ui-src/src/components/modals/CloseEntityModal.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import { axe } from "jest-axe";
+// components
+import { CloseEntityModal } from "components";
+// utils
+import { mockEntityDetailsOverlayVerbiage } from "utils/testing/setupJest";
+
+const mockCloseHandler = jest.fn();
+
+const modalComponent = (
+  <CloseEntityModal
+    verbiage={mockEntityDetailsOverlayVerbiage}
+    modalDisclosure={{
+      isOpen: true,
+      onClose: mockCloseHandler,
+    }}
+  />
+);
+
+const {
+  closeOutModalTitle,
+  closeOutModalBodyText,
+  closeOutModalConfirmButtonText,
+} = mockEntityDetailsOverlayVerbiage.closeOutModal;
+
+describe("Test CloseEntityModal", () => {
+  beforeEach(async () => {
+    await act(async () => {
+      await render(modalComponent);
+    });
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("CloseEntityModal shows the contents", () => {
+    expect(screen.getByText(closeOutModalTitle)).toBeTruthy();
+    expect(screen.getByText(closeOutModalBodyText)).toBeTruthy();
+    expect(screen.getByText(closeOutModalConfirmButtonText)).toBeTruthy();
+    expect(screen.getByText("Cancel")).toBeTruthy();
+  });
+
+  test("CloseEntityModal top close button can be clicked", () => {
+    fireEvent.click(screen.getByText("Close"));
```

Comment:
suggestion: use `userEvent` instead of `fireEvent` as per [recommendations from testing-library](https://testing-library.com/docs/user-event/intro/#differences-from-fireevent)

of very little consequence to this PR so I'm ok with it as is, just something for future consideration